src/helpers/__tests__/timers.test.ts

```diff
@@ -0,0 +1,8 @@
+import { jestFakeTimersAreEnabled } from '../timers';
+describe('timers', () => {
+  it('should not mock timers if SKIP_AUTO_FAKE_TIMERS_DETECTION is set', async () => {
+    process.env.SKIP_AUTO_FAKE_TIMERS_DETECTION = 'true';
```

Comment:
```suggestion
    process.env.RNTL_SKIP_AUTO_DETECT_FAKE_TIMERS = 'true';
```

Let's add `RNTL_` prefix as for `RNTL_SKIP_AUTO_CLEANUP`.

nit: I've also suggested `AUTO_DETECT_FAKE_TIMERS` vs `AUTO_FAKE_TIMER_DETECTION` as slightly more readable.

Reply:
done